java/ql/lib/semmle/code/java/frameworks/javaee/ejb/EJB.qll

```diff
@@ -33,14 +33,14 @@ class SessionEJB extends EJB {
     // Either the EJB does not declare any business interfaces explicitly
     // and implements a single interface candidate,
     // which is then considered to be the business interface...
-    count(this.getAnExplicitBusinessInterface()) = 0 and
-    count(this.getAnImplementedBusinessInterfaceCandidate()) = 1 and
+    not exists(this.getAnExplicitBusinessInterface()) and
+    exists(unique( | | this.getAnImplementedBusinessInterfaceCandidate())) and
```

Comment:
I remember @jbj tried to replace the C++ occurrences of `count(...) = 1` with `exists(unique(...))` and concluded that the generated RA was worse. The conversation about this has disappeared from Slack now, unfortunately. This was also _just_ after the `unique` aggregate was introduced, so the situation might be better now. It's worth checking up on, though!

Reply:
The Java evaluation is also not looking all that well.  
I'm waiting for some retries before I conclude anything, but I might end up reverting the Java changes.

Reply:
The C++ evaluation also looks weird, so I think I'll just do Python in this PR.